### DISCUSSION TITLE
GLUONTS: gave NaN loss and it will be ignored

### DISCUSSION THREAD

**Discussion by Deepankar18TSC** [awslabs/gluonts#1948] [Q&A]
Hi,

I am trying to train a model with below estimator values:
estimator = MyProbEstimator(
    prediction_length=24,
    context_length=48,
    freq='MS',
    distr_output=GaussianOutput(),
    num_cells=40,
    trainer=Trainer(
        ctx="cpu",
        epochs=5,
        learning_rate=1e-3,
        hybridize=False,
        num_batches_per_epoch=100
    )
)
""
__main__.MyProbEstimator(batch_size=32, context_length=48, distr_output=gluonts.mx.distribution.gaussian.GaussianOutput(), freq="MS", num_cells=40, num_sample_paths=100, prediction_length=24, scaling=True, trainer=gluonts.mx.trainer._base.Trainer(add_default_callbacks=True, batch_size=None, callbacks=None, clip_gradient=10.0, ctx=mxnet.context.Context("cpu", 0), epochs=5, hybridize=False, init="xavier", learning_rate=0.001, learning_rate_decay_factor=0.5, minimum_learning_rate=5e-05, num_batches_per_epoch=100, patience=10, weight_decay=1e-08))
[demand.xlsx](https://github.com/awslabs/gluon-ts/files/8501486/demand.xlsx)

""
When I use the estimator.train method, I am getting "gave NAN loss and it will be ignored"
Any help is highly appreciable.
